src/Boost.sol

```diff
@@ -146,15 +142,12 @@ contract Boost is IBoost, EIP712, Ownable, ERC721URIStorage {
         if (boost.end <= block.timestamp) revert BoostEnded();
         if (block.timestamp >= boost.start) revert ClaimingPeriodStarted();
 
-        uint256 balanceIncrease = 0;
-        if (tokenFee > MYRIAD) {
-            revert InvalidTokenFee();
-        } else {
-            // The token fee is calculated and subtracted from the deposit amount to get the boost balance increase
-            uint256 tokenFeeAmount = (_amount * tokenFee) / MYRIAD;
-            balanceIncrease = _amount - tokenFeeAmount;
-            tokenFeeBalances[address(boost.token)] += tokenFeeAmount;
-        }
+        // Using this non-intuitive computation to make it easier for the depositor to calculate the fee.
```

Comment:
could be nice to abstract these 4 lines and the same ones in `deposit` into a view function: 
`(balanceIncrease, tokenFeeAmount) = calculateFee(amount)`.